.travis.yml

```diff
@@ -2,9 +2,9 @@ language: swift
 xcode_workspace: Relisten.xcworkspace
 xcode_scheme: Relisten
 os: osx
-osx_image: xcode12.2
-xcode_scheme: Relisten
-install:
+osx_image: xcode12.1
+xcode_destination: platform=iOS Simulator,name=iPhone 12 Pro Max
```

Comment:
This isn't necessary since we're using a build script and it includes these properties. Adding them here will just create confusion if someone tries to edit them in the future, since they won't have any effect.

I'm in favor of just switching over to GitHub Actions though, so we can also leave this file as-is.